updater/lib/dependabot/sorbet/runtime.rb

```diff
@@ -0,0 +1,21 @@
+# typed: true
+# frozen_string_literal: true
+#
+require "sorbet-runtime"
+
+module Dependabot
+  module Sorbet
+    module Runtime
+      class InformationalError < StandardError; end
+
+      def self.silently_report_errors!
+        T::Configuration.call_validation_error_handler = lambda do |sig, opts|
+          error = InformationalError.new(opts[:pretty_message])
+          error.set_backtrace(caller.dup)
+
+          Sentry.capture_exception(error)
```

Comment:
Sorbet is getting confused between `Dependabot::Sentry` and `Sentry`.

```suggestion
          ::Sentry.capture_exception(error)
```